### PR TITLE
Fix unexpected exception return behavior

### PR DESCRIPTION
**Pull request by yenWu**
This #14 issue is found on other labs. I refer to #15 to fix the labs, "04-ContextSwitch-2" and "06-Preepmtive".

### CODE REVIEW

File: 06-Preemptive/context_switch.S [hunk 1]
```diff
@@ -22,14 +22,10 @@ activate:
...
 
-	/* switch to process stack */
-	msr psp, r0
-	mov r0, #3
-	msr control, r0
-	isb
-
 	/* load user state */
-	pop {r4, r5, r6, r7, r8, r9, r10, r11, lr}
+	ldmia r0!, {r4, r5, r6, r7, r8, r9, r10, r11, lr}
```
Comment (jserv) [inline 06-Preemptive/context_switch.S]: This patch was not intended to fix unexpected exception return behavior. Instead, it eliminates annoying checking that would trigger exception return or not and manipulate saving register on PSP directly.

Comment (yenWu) [inline 06-Preemptive/context_switch.S]: You are right! I would change the commit header.

Comment (yenWu) [inline 06-Preemptive/context_switch.S]: @jserv , BTW, should I new another pull request? It would not consist with the subject.

Comment (jserv) [inline 06-Preemptive/context_switch.S]: Use `git rebase -i` and `git commit --amend` to rework these commits.

**Pull request merged**